code/edgeidentity/src/androidTest/java/com/adobe/marketing/mobile/edge/identity/IdentityAndroidTestUtil.java

```diff
@@ -0,0 +1,303 @@
+/*
+  Copyright 2021 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.edge.identity;
```

Comment:
I have a proposal, should we write all our new files in kotlin? This util is written already and we should not rewrite this but do you guys think it is a good idea if we start writing all the new code/files in kotlin?
@prudrabhat @emdobrin @kevinlind

Reply:
It will help us write the files faster and also less boilerplate code especially for test helpers.

Reply:
During Core 2.0 development we found that writing new code in kotlin helped a great deal in improving code flow and and the amount of code written.  Configuration and Signal were re-written in kotlin entirely!